packages/cerebral-forms/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Signals, actions and state factories to create forms",
   "main": "lib/index.js",
   "scripts": {
-    "test": "../../node_modules/.bin/mocha --compilers js:../../node_modules/babel-register 'src/**/*.test.js'",
+    "test": "node ../../node_modules/mocha/bin/mocha --compilers js:../../node_modules/babel-register 'src/**/*.test.js'",
```

Comment:
@henri-hulski  yes its there but  windows can't run the script there.
imo its because of two different reasons:
1. windows uses backslash in path
2. i have never seen such kind of scripts for windows (.bin/mocha). looks like its more like a unix thing to me

but maybe i'm just missing something for my dev environment :)